src/reconcile/rec_write.c

```diff
@@ -6591,3 +6594,37 @@ __rec_dictionary_lookup(
 	*dpp = next;
 	return (0);
 }
+
+/*
+ * __verbose_lookaside_write --
+ *	Create a verbose message to display with details about the transaction
+ * state when performing a lookaside table write.
```

Comment:
The examples I found of multi-line function level comments re-indent to the same level, for example:
https://github.com/wiredtiger/wiredtiger/blob/develop/src/txn/txn_ckpt.c#L1104

Having looked a bit further, it seems we aren't consistent, some places do that and others don't.